cluster-management/debugging/manual-rollbacks/index.md

```diff
@@ -0,0 +1,185 @@
+---
+layout: docs
+slug: guides
+title: Manual CoreOS Rollbacks
+category: cluster_management
+sub_category: debugging
+weight: 5
+---
+
+# Performing Manual CoreOS Rollbacks
+
+As much as we all love flawless, automatic updates, there may be occasions when
```

Comment:
I dunno. "Seamless automatic updates are a key feature of CoreOS, and most issues <something about rolling forward>. On occasion, however, it may be desirable to manually roll back an update. The procedure to do this is straightforward and can be achieved with the commands described below."

or: [1011100111001001111001](https://www.youtube.com/watch?v=B1BdQcJ2ZYY)